images/linux/scripts/installers/basic.sh

```diff
@@ -45,114 +77,9 @@ fi
-
-echo "Install locales"
-apt-get install -y --no-install-recommends locales
-
-echo "Install netcat"
-apt-get install -y --no-install-recommends netcat
-
-echo "Install openssh-client"
-apt-get install -y --no-install-recommends openssh-client
-
-echo "Install rsync"
-apt-get install -y --no-install-recommends rsync
-
-echo "Install shellcheck"
-apt-get install -y --no-install-recommends shellcheck
-
-echo "Install sudo"
-apt-get install -y --no-install-recommends sudo
-
-echo "Install telnet"
-apt-get install -y --no-install-recommends telnet
-
-echo "Install time"
-apt-get install -y --no-install-recommends time
-
-echo "Install tzdata"
-apt-get install -y --no-install-recommends tzdata
-
-echo "Install unzip"
-apt-get install -y --no-install-recommends unzip
-
-echo "Install upx"
-apt-get install -y --no-install-recommends upx
-
-echo "Install wget"
-apt-get install -y --no-install-recommends wget
-
-echo "Install zip"
-apt-get install -y --no-install-recommends zip
-
-echo "Install zstd"
-apt-get install -y --no-install-recommends zstd
-
-echo "Install libxkbfile"
-apt-get install -y --no-install-recommends libxkbfile-dev
-
-echo "Install pkg-config"
-apt-get install -y --no-install-recommends pkg-config
-
-echo "Install libsecret-1-dev"
-apt-get install -y --no-install-recommends libsecret-1-dev
-
-echo "Install libxss1"
-apt-get install -y --no-install-recommends libxss1
-
-echo "Install libgconf-2-4"
-apt-get install -y --no-install-recommends libgconf-2-4
-
-echo "Install dbus"
-apt-get install -y --no-install-recommends dbus
-
-echo "Install xvfb"
-apt-get install -y --no-install-recommends xvfb
-
-echo "Install libgbm-dev"
-apt-get install -y --no-install-recommends libgbm-dev
-
-echo "Install libgtk"
-apt-get install -y --no-install-recommends libgtk-3-0
-
-echo "Install tk"
-apt install -y tk
-
-echo "Install fakeroot"
-apt-get install -y --no-install-recommends fakeroot
-
-echo "Install dpkg"
-apt-get install -y --no-install-recommends dpkg
-
-echo "Install rpm"
-apt-get install -y --no-install-recommends rpm
-
-echo "Install xz-utils"
-apt-get install -y --no-install-recommends xz-utils
-
-echo "Install xorriso"
-apt-get install -y --no-install-recommends xorriso
-
-echo "Install zsync"
-apt-get install -y --no-install-recommends zsync
-
-echo "Install curl"
-apt-get install -y --no-install-recommends curl
-
-echo "Install parallel"
-apt-get install -y --no-install-recommends parallel
-
-echo "Install gnupg2"
-apt-get install -y --no-install-recommends gnupg2
-
-echo "Install lib32z1"
-apt-get install -y --no-install-recommends lib32z1
-
 # Run tests to determine that the software installed as expected
 echo "Testing to make sure that script performed as expected, and basic scenarios work"
-for cmd in curl file ftp jq netcat ssh parallel rsync shellcheck sudo telnet time unzip wget zip; do
+for cmd in curl file ftp jq netcat ssh parallel rsync shellcheck sudo telnet time unzip wget zip m4 bison flex texinfo; do
```

Comment:
Is it possible to use to use array `common_packages` here?
And at the bottom with `DocumentInstalledItemIndent`.

Reply:
Done.